### PR TITLE
Add task list feature for the welcome page

### PR DESCRIPTION
**Pull request by xinyi98**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "prettier": "^2.2.1",
-    "gh-pages": "^2.2.0"
+    "gh-pages": "^2.2.0",
+    "prettier": "^2.2.1"
```
Comment (franc-hoon) [inline package.json]: What's this for?

File: src/assets/icons/PlusIcon.jsx [hunk 2]
```diff
@@ -9,6 +10,7 @@ function PlusIcon() {
       y="0px"
       viewBox="0 0 455 455"
       style={{height: "50%", fill: "white"}}
+      onClick={handleOnClick}
```
Comment (franc-hoon) [inline src/assets/icons/PlusIcon.jsx]: Why are we adding on clicks to icons?

File: src/components/global/TaskList.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <div className="all-tasks">
+            {tasks.map((t) => (
+                <div key={t.Task_id}>
+                    <Task name={t.Name} />
+                </div>
+            ))}
+            </div>
+
+            <div className="add-task-button-container">
+            <NewTask />
```
Comment (franc-hoon) [inline src/components/global/TaskList.jsx]: Indent, run prettier

File: src/pages/homeView.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ import HomepageImage from "../assets/images/HomepageImage";
 import DateTime from "../components/DateTime";
 import Navbar from '../components/Navbar';
 
+
```
Comment (franc-hoon) [inline src/pages/homeView.js]: Remove extra line

File: src/stylesheets/components/global/taskList.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+::-webkit-scrollbar {
+    width: 0;
+    background: transparent;
+}
+
+.add-task-button-container{
+    width: 10vw;
+    display: flex; flex:1
```
Comment (franc-hoon) [inline src/stylesheets/components/global/taskList.scss]: Fix

**Review by franc-hoon — CHANGES_REQUESTED** [review]
Just a few minor changes

**Comment by xinyi98** [comment]
> Just a few minor changes

Finished ;

File: src/stylesheets/components/global/taskList.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
+.task-list {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+}
+
+.all-tasks {
+    width: 22vw;
+    height: 80vh;
+    background-color: #2e2d40;
```
Comment (franc-hoon) [inline src/stylesheets/components/global/taskList.scss]: there's a variable for this. can be replaced with `$solid-background`

**Review by franc-hoon — APPROVED** [review]
LGTM, nice work 👍

**Pull request merged**